.vscode/launch.json

```diff
@@ -14,8 +14,8 @@
             ],
             "program": "${workspaceFolder}/.compiledjs/create-note.js",
             "args": [
-                "--drawinguri=https://cad.onshape.com/documents/854e914aa52304e246a9eec9/w/abfdc05bfa1f41beddef6799/e/0bb00c2b35c0dad605592265",
```

Comment:
It is good to define a variable for the server base.

Reply:
Could you explain in a bit more detail what you mean?  I was following the pattern that Raj created in his onshape-ts-client repo - https://github.com/onshape-public/onshape-ts-client/blob/master/.vscode/launch.json

But I'm happy to improve this, once I understand what you mean.  Thanks.

Reply:
I will leave this improvement to you.  Thanks.